runtime/compiler/CMakeLists.txt

```diff
@@ -349,7 +371,11 @@ target_link_libraries(j9jit
 )
 
 if(JITSERVER_SUPPORT)
-	target_link_libraries(j9jit PRIVATE ${PROTOBUF_LIBRARY})
+	if(JITSERVER_ENABLE_SSL)
+		target_link_libraries(j9jit PRIVATE ${PROTOBUF_LIBRARY} ${OPENSSL_SSL_LIBRARY} ${OPENSSL_CRYPTO_LIBRARY})
+	else()
+		target_link_libraries(j9jit PRIVATE ${PROTOBUF_LIBRARY})
```

Comment:
I think the original `target_link_libraries(j9jit PRIVATE ${PROTOBUF_LIBRARY})` should be left alone; just add the SSL libraries if necessary.